### PR TITLE
Remove Demeteorizer

### LINKED ISSUE #63: Avoid "output directory is under your source tree" Meteor warning
Bundle the project to a temp directory rather than the local directory before deploying.

### PR DESCRIPTION
**Pull request by harlanj**
- Support Meteor uploads via CLI without the use of demeteorizer
- Add .meteor/local to ignore
- Remove demeteorizer from dependencies

Closes #63 #102 #72 #99 #66 #101

### CODE REVIEW

**Comment by harlanj** [comment]
I can see [this block of code](https://github.com/onmodulus/modulus-cli/blob/master/lib/commands/project.js#L619) being a problem with deploys. Any input? It forces another deploy over a project that's in the UPLOADING state.

File: lib/controllers/project.js [hunk 1]
```diff
@@ -221,40 +220,10 @@ Project.prototype.deploy = function(projectId, dir, includeModules, registry, me
...
-
-      function (out, cb) {
-        var compressTarget = projectType === Project.types.METEOR
-          ? path.join(out, 'bundle') : out;
-
+      function (cb) {
         modulus.io.print('Compressing project...');
         projectController.packageProject(
-          compressTarget,
+          dir,
```
Comment (franvarney) [inline lib/controllers/project.js]: Where is this coming from?

**Comment by franvarney** [comment]
:+1:

**Comment by jackboberg** [comment]
Let's have that ^^ conversation in a separate thread.

**Comment by jackboberg** [comment]
waiting on some internal releases, and I will publish this as `6.0.0` under a npm `next` dist-tag for the weekend.

**Pull request merged**